src/tasks.scala

```diff
@@ -260,6 +260,38 @@ object Tasks {
     }
   }
 
+  val checkAarsTaskDef = Def.task {
+    implicit val log = streams.value.log
+    implicit val struct = buildStructure.value
+    val deps = thisProjectRef.value.deepDeps
+    def revDependents(aar: ModuleID) = deps.filter(_.dependsOnAar(aar))
+    def pkgDependents(data: (String, Seq[ModuleID])) = {
+      (data._1, data._2, data._2 map(revDependents))
+    }
+    thisProjectRef.value
+      .deepAarDeps
```

Comment:
dunno how you'll get around this, probably need to depend on `transitiveAars` at this point